person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    @age >= 18 or parent_permission == true
```

Comment:
- [ ] Kindly change the implementation to match this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Public%20method%20can_use_services%3F%20that%20returns%20true%20if%20person%20is%20of%20age%20or%20if%20they%20have%20permission%20from%20parents.) by using the private method `of_age?` instead of the `@age>=18`.

Reply:
Thanks